analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -181,55 +181,47 @@ class Gradle(name: String, analyzerConfig: AnalyzerConfiguration, repoConfig: Re
-        // Only look for a package when there was no error resolving the dependency.
-        if (dependency.error == null) {
+        // Only look for a package if there was no error resolving the dependency and it is no project dependency.
+        if (dependency.error == null && dependency.localPath == null) {
             val identifier = "${dependency.groupId}:${dependency.artifactId}:${dependency.version}"
 
-            val rawPackage by lazy {
-                Package(
-                        id = Identifier(
-                                type = "Maven",
-                                namespace = dependency.groupId,
-                                name = dependency.artifactId,
-                                version = dependency.version
-                        ),
-                        declaredLicenses = sortedSetOf(),
-                        description = "",
-                        homepageUrl = "",
-                        binaryArtifact = RemoteArtifact.EMPTY,
-                        sourceArtifact = RemoteArtifact.EMPTY,
-                        vcs = VcsInfo.EMPTY
-                )
-            }
-
             packages.getOrPut(identifier) {
-                val pkg = if (dependency.pomFile != null) {
-                    val artifact = DefaultArtifact(dependency.groupId, dependency.artifactId, dependency.classifier,
-                            dependency.extension, dependency.version)
-                    try {
-                        maven.parsePackage(artifact, repositories)
-                    } catch (e: ProjectBuildingException) {
-                        e.showStackTrace()
-
-                        log.error {
-                            "Could not get package information for dependency '$identifier': ${e.message}"
-                        }
+                val artifact = DefaultArtifact(dependency.groupId, dependency.artifactId, dependency.classifier,
+                        dependency.extension, dependency.version)
 
-                        issues += OrtIssue(source = managerName, message = e.collectMessagesAsString())
+                try {
+                    maven.parsePackage(artifact, repositories)
+                } catch (e: ProjectBuildingException) {
+                    e.showStackTrace()
 
-                        rawPackage
+                    log.error {
+                        "Could not get package information for dependency '$identifier': ${e.message}"
                     }
-                } else {
-                    rawPackage
-                }
 
-                pkg.copy(vcsProcessed = dependency.localPath?.let { processProjectVcs(File(it), pkg.vcs) }
-                        ?: processPackageVcs(pkg.vcs))
+                    issues += OrtIssue(source = managerName, message = e.collectMessagesAsString())
+
+                    Package(
+                            id = Identifier(
+                                    type = dependency.pomFile?.let { "Maven" } ?: "Unknown",
```

Comment:
I don't have a concrete example where resolution fails for a non-Maven dependency. But my thinking came exactly from our test cases: There resolution (deliberately) fails because we do not define any repository. If no repository is defined, you cannot even tell for sure that the dependency is a *Maven* artifact.

Reply:
Except if it is the transitive dependency of another Maven artifact.

Reply:
I was assuming there was a (standard) way e.g. via a plugin to make Maven understand Ivy repositories, but you're right, there is not.

Reply:
I just realized a mistake of mine: I though this code path would also get executed if the error is something else than `ProjectBuildingException`. But it *only* gets executed if an `ProjectBuildingException` occurred, which is an exception from Maven code, and as you correctly pointed out Maven projects can only have Maven-style dependencies, it's actually also correct to hard-code "Maven" as the package type here.